src/Ocelot/Infrastructure/JsonSerializerOptionsFactory.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.Unicode;
+
+namespace Ocelot.Infrastructure;
+
+public static class JsonSerializerOptionsFactory
```

Comment:
Why `Factory` for static definitions? In terms of .NET, factory is
- delegate: `Func{T}` or other custom ones
- class which produces concrete ones based on abstract interface
- method which produces concrete classes based on abstract interface 
- a dynamic building functionality

and... seems renaming to short `JsonSerializerOptions` is impossible because of returning type, so the best name could be `OcelotSerializerOptions`.

Reply:
done